v03_pipeline/lib/reference_data/config.py

```diff
@@ -503,9 +503,9 @@ def custom_mpc_select(ht):
     },
     'mitimpact': {
         '38': {
-            'version': '3.0.7',
-            'path': 'gs://seqr-reference-data/GRCh38/mitochondrial/MitImpact/MitImpact_db_3.0.7.ht',
-            'select': {'score': 'APOGEE_score'},
+            'version': '3.1.3',
+            'path': 'gs://seqr-reference-data/GRCh38/mitochondrial/MitImpact/MitImpact_db_3.1.3.ht',
```

Comment:
did we have to change anything in an old import script to generate the ht?

Reply:
not really, I followed the same procedure of reading the .txt file into a hail table, annotating locus and alleles, and the only difference was that I had to annotate both APOGEE1_score and APOGEE2_score as floats

Reply:
cool... I guess we should update that script 🤷?  Better to have whatever we did codified even if it's rough.